src/Tests/Unit/BaseIsracardAmexBranches.test.ts

```
@@ -0,0 +1,314 @@
+    expect(page.on).toHaveBeenCalledWith('response', anyFn);
+  });
+
+  it('logs ProxyRequestHandler URLs via response callback', async () => {
+    mockValidate('1');
+    mockLogin('1');
+    mockAccounts();
+    mockTxns();
+    const page = CREATE_MOCK_PAGE();
+    MOCK_CONTEXT.newPage.mockResolvedValue(page);
+    await new TestAmexScraper().scrape(CREDS);
+    type ResponseHandler = (_r: { url: () => string; status: () => number }) => string;
```

ESLint

[error] '_r' is defined but never used.

(no-unused-vars)